contrib/sles11/alarm-notify-basic.bash3.sh

```diff
@@ -463,31 +457,31 @@ send_syslog() {
         temp1=''
         temp2=''
 
-        prefix=$(echo ${target} | cut -d '/' -f 2)
-        temp1=$(echo ${target} | cut -d '/' -f 1)
+        prefix=$(echo "${target}" | cut -d '/' -f 2)
+        temp1=$(echo "${target}" | cut -d '/' -f 1)
 
-        if [[ ${prefix} != ${temp1} ]] ; then
-            if (echo ${temp1} | grep -q '@' ) ; then
-                temp2=$(echo ${temp1} | cut -d '@' -f 1)
-                host=$(echo ${temp1} | cut -d '@' -f 2)
+        if [[ "${prefix}" != "${temp1}" ]] ; then
+            if (echo "${temp1}" | grep -q '@' ) ; then
+                temp2=$(echo "${temp1}" | cut -d '@' -f 1)
+                host=$(echo "${temp1}" | cut -d '@' -f 2)
 
-                if [ ${temp2} != ${host} ] ; then
-                    priority=${temp2}
+                if [ "${temp2}" != "${host}" ] ; then
```

Comment:
Can we omit the space preceding ; as we do in other places?

```suggestion
                if [ "${temp2}" != "${host}" ]; then
```